### ISSUE TITLE
[Error in plugin 'gulp-stylus'] failed to locate @import file ...

### ISSUE THREAD

**Issue by dereklin** [Swiip/generator-gulp-angular#569]
So I manually downloaded gulp-angular generator and installed it.  Created a new app.  Then I ran the gulp command and saw this problem:  (as of May 29, 2015):

```
[12:43:44] [Stylus] Error in plugin 'gulp-stylus'
Message:
    c:\Temp\test6\src\app\index.styl:18:9
   14|  *  Do not remove this comments bellow. It's the markers used by gulp-inj
ect to inject
   15|  *  all your stylus files automatically
   16|  */
   17| // injector
   18| // endinjector
---------------^
   19|

failed to locate @import file src/app/components/malarkey/malarkey.styl

Details:
    lineno: 18
    column: 9
    filename: c:\Temp\test6\src\app\index.styl
    stylusStack:
    input: .browsehappy
  margin 0.2em 0
  background #ccc
  color #000
  padding 0.2em 0

.thumbnail
  height 200px

  img.pull-right
    width 50px

/**
 *  Do not remove this comments bellow. It's the markers used by gulp-inject to
inject
 *  all your stylus files automatically
 */
// injector
// endinjector
```

**Comment by zckrs** [comment]
What command you run ?

**Comment by dereklin** [comment]
gulp

**Comment by zckrs** [comment]
Can you paste your yo-rc ?

**Comment by zckrs** [comment]
Not reproduce. All work fine with stylus. If you don't paste your `.yo-rc.json` I can't be sure I use identical config.

**Comment by dereklin** [comment]
Please note, I downloaded the the zip file from github and manually installed the gulp-angular.  Here is the yo-rc:
```
{
  "generator-gulp-angular": {
    "version": "0.11.0",
    "props": {
      "angularVersion": "~1.3.15",
      "angularModules": [
        {
          "key": "animate",
          "module": "ngAnimate"
        }
      ],
      "jQuery": {
        "key": "jquery2"
      },
      "resource": {
        "key": "none",
        "module": null
      },
      "router": {
        "key": "ui-router",
        "module": "ui.router"
      },
      "ui": {
        "key": "bootstrap",
        "module": null
      },
      "bootstrapComponents": {
        "key": "ui-bootstrap",
        "module": "ui.bootstrap"
      },
      "cssPreprocessor": {
        "key": "stylus",
        "extension": "styl"
      },
      "jsPreprocessor": {
        "key": "none",
        "extension": "js",
        "srcExtension": "js"
      },
      "htmlPreprocessor": {
        "key": "none",
        "extension": "html"
      },
      "foundationComponents": {
        "name": null,
        "version": null,
        "key": null,
        "module": null
      },
      "paths": {
        "src": "src",
        "dist": "dist",
        "e2e": "e2e",
        "tmp": ".tmp"
      }
    }
  }
}
```

**Comment by zckrs** [comment]
Always not reproduce can you try to clone master and use npm  link ?

**Comment by dereklin** [comment]
What do you mean by using npm link?

I git clone the master, and then I encountered this error:

```
C:\Temp\test10>yo gulp-angular
module.js:338
    throw err;
          ^
Error: Cannot find module 'yeoman-generator'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
```

When I do npm install -g generator-gulp-angular, it installs the "old" version.  And the "old" version works.  

This problem is not a high priority for me.  You can close it for now if you can't reproduce it.

**Comment by zckrs** [comment]
`$ git clone git@github.com:Swiip/generator-gulp-angular.git`
Go into folder then:
`$ npm install`
`$ npm link`
Go into your newProject folder:
`$ yo gulp-angular`

**Comment by dereklin** [comment]
I was not able to do npm link -- not sure if it's related to firewall issue.  However, I was able to run yo gulp-angular.  But I am experiencing the same problem:

```
C:\Temp\test9>gulp
[08:59:17] Using gulpfile C:\Temp\test9\gulpfile.js
[08:59:17] Starting 'clean'...
[08:59:17] Finished 'clean' after 66 ms
[08:59:17] Starting 'default'...
[08:59:17] Starting 'scripts'...
[08:59:17] Starting 'styles'...
[08:59:18] Starting 'partials'...
[08:59:18] Starting 'fonts'...
[08:59:18] Starting 'other'...
[08:59:18] Finished 'default' after 1.37 s
[08:59:18] gulp-inject 2 files into index.styl.
[08:59:18] [Stylus] Error in plugin 'gulp-stylus'
Message:
    C:\Temp\test9\src\app\index.styl:18:9
   14|  *  Do not remove this comments bellow. It's the markers used by gulp-inj
ect to inject
   15|  *  all your stylus files automatically
   16|  */
   17| // injector
   18| // endinjector
---------------^
   19|

failed to locate @import file src/app/components/navbar/navbar.styl

Details:
    lineno: 18
    column: 9
    filename: C:\Temp\test9\src\app\index.styl
    stylusStack:
    input: .browsehappy
  margin 0.2em 0
  background #ccc
  color #000
  padding 0.2em 0

.thumbnail
  height 200px

  img.pull-right
    width 50px

/**
 *  Do not remove this comments bellow. It's the markers used by gulp-inject to
inject
 *  all your stylus files automatically
 */
// injector
// endinjector

[08:59:18] Finished 'styles' after 1.04 s
[08:59:19] Finished 'partials' after 719 ms
[08:59:19] all files 8.22 kB
[08:59:19] Finished 'scripts' after 1.76 s
[08:59:19] Starting 'inject'...
[08:59:19] Finished 'fonts' after 568 ms
[08:59:19] gulp-inject Nothing to inject into index.html.
[08:59:19] gulp-inject 10 files into index.html.
[08:59:19] Finished 'inject' after 123 ms
[08:59:19] Starting 'html'...
[08:59:19] gulp-inject 1 files into index.html.
[08:59:20] Finished 'other' after 1.86 s
[08:59:29] 'dist\' styles\vendor-36471da4.css 56.9 kB
[08:59:29] 'dist\' scripts\app-f8cb8d88.js 6.32 kB
[08:59:29] 'dist\' scripts\vendor-85f11d16.js 367.45 kB
[08:59:29] 'dist\' index.html 992 B
[08:59:29] 'dist\' all files 431.67 kB
[08:59:29] Finished 'html' after 9.96 s
[08:59:29] Starting 'build'...
[08:59:29] Finished 'build' after 3.53 µs
```

**Comment by Swiip** [comment]
Not reproduced either. Your stylus compile is stange, it points on a comment...

What do you mean by npm init failed ? Can you delete the `node_modules` folder from both the generator and the generated project and run npm installs again?

**Comment by syzer** [comment]
+1

**Comment by zckrs** [comment]
@syzer thank you for this necessary comment who we help to understand origin of issue

**Comment by syzer** [comment]
i also started new issue... when u do clean you setup on mac it does not run the stylus task!

**Comment by syzer** [comment]
#592

**Comment by zckrs** [comment]
:tada: Just released v0.12.0 :tada: can you try with last version ? 

I close if you issue already with v0.12.0 can you open a new issue ?

**Comment by julkue** [comment]
I still have this issue, but simply when using gulp-stylus.
